### PR TITLE
Image Classification

### PR DESCRIPTION
**Pull request by zhangabner**
update with MLflow

### CODE REVIEW

**Comment by mlaradji** [comment]
Hi @zhangabner. Thank you for your pull request!

Could you add `keras` and any other dependencies to `conda.yaml`? When I ran `mlflow run .`, I get the following output:
```
2019/09/10 03:02:03 INFO mlflow.projects: === Created directory /tmp/tmp53gb99ro for downloading remote URIs passed to arguments of type 'path' ===
2019/09/10 03:02:03 INFO mlflow.projects: === Running command 'source activate mlflow-c9bbec7a7e8c34bd7529f4c3e441aadc335f2fc4 1>&2 && python image_classification.py 300 ' in run with ID '8114d06e2c534a57a14bb8298f13be26' === 
Traceback (most recent call last):
  File "image_classification.py", line 6, in <module>
    from keras.preprocessing.image import ImageDataGenerator
ModuleNotFoundError: No module named 'keras'
2019/09/10 03:02:04 ERROR mlflow.cli: === Run (ID '8114d06e2c534a57a14bb8298f13be26') failed ===
```

File: image_classification.py [hunk 1]
```diff
@@ -0,0 +1,559 @@
...
+    label_predict = np.argmax(score_predict)
+    predicted.append(str(label_predict))
+
+
+# In[20]:
+
+submit['diagnosis'] = predicted
+submit.to_csv('submission.csv', index=False)
+submit.head()
+
```
Comment (slavakurilyak) [inline image_classification.py]: @zhangabner Lines 533-559 (image_classification.py) are different from Lines 458-464 (tasks/computer-vision/image-classification/image_classification.py). Was this intentional?

**Comment by slavakurilyak** [comment]
@zhangabner Please remove your `image_classification.py` and keep your `tasks/computer-vision/image-classification/image_classification.py`

File: image_classification.py (hunk 1 above)
Comment (zhangabner) [reply image_classification.py]: Yes, I use mlflow to track the training, not to inference yet.

**Comment by zhangabner** [comment]
done

Slava Kurilyak <notifications@github.com> 于2019年9月14日周六 上午8:46写道：

**Pull request closed without merging**